src/models/player.rs

```diff
@@ -1077,28 +1099,7 @@ fn library_item_update<E: Env + 'static>(
         }
         _ => None,
     };
-    if *library_item != next_library_item {
```

Comment:
I suppose this part was moved above with the `LibraryChanged` internal message and that's why you've removed the logic here?

Reply:
I do not really understand what was the purpose of this code because instead of updating the library item inside the model, it just dispatch the internal message to update it in the library
We don't want that either, we want to update the library item from the model after it was updated from the library